dubbo-rpc/dubbo-rpc-triple/src/main/java/org/apache/dubbo/rpc/protocol/tri/TripleExceptionWrapperUtils.java

```diff
@@ -0,0 +1,131 @@
+package org.apache.dubbo.rpc.protocol.tri;
```

Comment:
Please add ASF license header for all newly created files

Reply:
> Please add ASF license header for all newly created files

Maybe we can use apache/skywalking-eyes/header@main to automatically add ASF license header to newly created files in GitHub action. I'm willing to submit a PR if needed, thanks.

Reply:
> > Please add ASF license header for all newly created files
> 
> Maybe we can use apache/skywalking-eyes/header@main to automatically add ASF license header to newly created files in GitHub action. I'm willing to submit a PR if needed, thanks.

@sunchaser-lilu go ahead~